modules/math/src/main/java/com/opengamma/strata/math/impl/interpolation/Interpolator1DFactory.java

```diff
@@ -232,16 +229,16 @@ public static String getInterpolatorName(final Interpolator1D interpolator) {
     return s_instanceNames.get(interpolator.getClass());
   }
 
-  public static CurveInterpolator findCurveInterpolator(final String interpolatorName) {
+  public static Interpolator1D findCurveInterpolator(final String interpolatorName) {
```

Comment:
The name of this method clashes with the convention that `find*` should return an `Optional` and `get*` should throw an exception if the data isn't found.